components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/SAMLAssertionClaimsCallbackTest.java

```diff
@@ -71,34 +86,42 @@
         AuthorizationGrantCache.class,
         PrivilegedCarbonContext.class,
         IdentityTenantUtil.class,
-        OAuth2ServiceComponentHolder.class
+        OAuth2ServiceComponentHolder.class,
+        ClaimMetadataHandler.class,
+        UserCoreUtil.class
 })
-public class SAMLAssertionClaimsCallbackTest extends IdentityBaseTest {
-
-    private String SAMPLE_ACCESS_TOKEN = "4952b467-86b2-31df-b63c-0bf25cec4f86";
-    private int SAMPLE_TENANT_ID = 1234;
+public class SAMLAssertionClaimsCallbackTest {
+
+    private static final String SAMPLE_ACCESS_TOKEN = "4952b467-86b2-31df-b63c-0bf25cec4f86";
+    private static final int SAMPLE_TENANT_ID = 1234;
+    private static final String SAMPLE_TENANT_DOMAIN = "dummy_domain";
+    private static final String SAMPLE_CLIENT_ID = "u5FIfG5xzLvBGiamoAYzzcqpBqga";
+    private static final String SAMPLE_SERVICE_PROVIDER = "sampleSP";
+    private static final String carbonHome = Paths.get(System.getProperty("user.dir"),
```

Comment:
Set carbon home to target if you are not explictly reading configurations from carbon.xml

Reply:
carbon.xml is available in the resources and reading from there.

Reply:
Does it require for your test ? What happens if you set carbon home to target ?

Reply:
Initially, I used "resources" instead of "target", then got a file not found for carbon.xml. Then I gave the path to carbon.xml since it was there in the resources.
Anyway with "target" also it's working. Seems like it's not explicitly reading from the carbon.xml